.haml-lint_todo.yml

```diff
@@ -0,0 +1,114 @@
+# This configuration was generated by
+# `haml-lint --auto-gen-config`
+# on 2023-02-11 02:16:03 -0500 using Haml-Lint version 0.45.0.
+# The point is for the user to remove these configuration records
+# one by one as the lints are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of Haml-Lint, may require this file to be generated again.
+
+linters:
+  # Offense count: 76
+  RuboCop:
+    exclude:
+      - 'app/views/accounts/_og.html.haml'
+      - 'app/views/admin/account_actions/new.html.haml'
+      - 'app/views/admin/account_warnings/_account_warning.html.haml'
+      - 'app/views/admin/accounts/index.html.haml'
+      - 'app/views/admin/accounts/show.html.haml'
+      - 'app/views/admin/action_logs/index.html.haml'
+      - 'app/views/admin/announcements/edit.html.haml'
+      - 'app/views/admin/announcements/new.html.haml'
+      - 'app/views/admin/change_emails/show.html.haml'
+      - 'app/views/admin/custom_emojis/index.html.haml'
+      - 'app/views/admin/dashboard/index.html.haml'
+      - 'app/views/admin/disputes/appeals/_appeal.html.haml'
+      - 'app/views/admin/reports/actions/preview.html.haml'
+      - 'app/views/admin/reports/index.html.haml'
+      - 'app/views/admin/reports/show.html.haml'
+      - 'app/views/admin/roles/_form.html.haml'
+      - 'app/views/admin/statuses/show.html.haml'
+      - 'app/views/application/_card.html.haml'
+      - 'app/views/application/_sidebar.html.haml'
+      - 'app/views/auth/registrations/_sessions.html.haml'
+      - 'app/views/auth/registrations/new.html.haml'
+      - 'app/views/auth/sessions/new.html.haml'
+      - 'app/views/disputes/strikes/show.html.haml'
+      - 'app/views/layouts/application.html.haml'
+      - 'app/views/layouts/error.html.haml'
+      - 'app/views/layouts/mailer.html.haml'
+      - 'app/views/notification_mailer/_status.html.haml'
+      - 'app/views/notification_mailer/favourite.html.haml'
+      - 'app/views/notification_mailer/follow_request.html.haml'
+      - 'app/views/settings/applications/_fields.html.haml'
+      - 'app/views/settings/login_activities/_login_activity.html.haml'
+      - 'app/views/settings/preferences/appearance/show.html.haml'
+      - 'app/views/statuses/_detailed_status.html.haml'
+      - 'app/views/statuses/_og_image.html.haml'
+      - 'app/views/statuses/_poll.html.haml'
+      - 'app/views/statuses/_status.html.haml'
+      - 'app/views/statuses/show.html.haml'
+      - 'app/views/user_mailer/suspicious_sign_in.html.haml'
+      - 'app/views/user_mailer/warning.html.haml'
+      - 'app/views/user_mailer/webauthn_credential_added.html.haml'
+      - 'app/views/user_mailer/webauthn_credential_deleted.html.haml'
+
+  # Offense count: 918
+  LineLength:
```

Comment:
May be better to just move this from the TODO to a regular disabled rule if it's unlikely to be fixed